PEPRest/src/main/resources/application.properties

```diff
@@ -5,11 +5,11 @@ logging.level.org.eclipse.jetty.util=INFO
 
 server.port=9999
 
-pep.id=0
+pep.id=1
 pep.uri=http://localhost:9999
 pep.revoke-type=HARD
-pep.policy-path=policies/PolicyWatch.xml
-pep.request-path=requests/Request.xml
+pep.policy-path=../res/xmls/policy-watch.xml
```

Comment:
Why ../res/ and not resources?
Also why ../ to move up?

Reply:
We have already have a doc/ and now maybe a res/ folder in the root of the repository.
In this way we have a single folder where we find all the xmls and pip txt files.
this xml can be reused by other projects.
This has been made because the split of UCSRest and UCSCore made me duplicate all those resources

Reply:
OK. Can we not define the  ../res/xmls part in the code? (see also my reply below)

Reply:
yes that's a really good idea.
It's because in some parts we didn't use the variables taken from the test application.properties

Reply:
@tezcan10 what do you mean by "Can we not define the ../res/xmls part in the code"?